### DISCUSSION TITLE
Student developer pack benefits not granted

### DISCUSSION THREAD

**Discussion by johnolawoyin14** [community/community#143106] [GitHub Education]
Good day, GitHub 
I applied for the student developer pack and it has been approved since a few days ago but I haven't received my benefits. Kindly help me review it. Thank you.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @johnolawoyin14! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by johnolawoyin14** [comment]
Thanks

**Comment by davevad93** [comment]
Hi @johnolawoyin14 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @johnolawoyin14 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
